pages/blog/whats-new-in-graphql-cli-4.1.0.mdx

```diff
@@ -1,18 +1,26 @@
 import withArticle from '../../ui/blog/article';
 
 export const meta = {
-  title: "What's new in GraphQL CLI 4.1.0",
+  title: "What's new in GraphQL CLI 4.1",
   author: 'ephelan',
-  tags: ['react'],
+  tags: [
+    'graphql',
+    'node',
+    'codegen',
+    'graphql-cli',
+    'graphql-config',
+    'graphback',
+  ],
   date: '2020-10-12',
-  description: "An overview of what we've added in this release of GraphQL CLI",
+  description: 'GraphQL CLI 4.1 is here!',
```

Comment:
Maybe for better SEO we should be more descriptive here?

Reply:
GraphQL CLI - modern way to redefine the way you work with GraphQL on client and server

Reply:
looks good :)